### PR TITLE
Added retry popover tests for system details and minor text fix

### PR DESCRIPTION
**Pull request by de1987**
# Description

- Added eslint file for cypress tests
- Added retry pop tests for system details
- Changed service timed out description as described in the mockups 
- Added ids for status and retry popover components
- Added fixtures for cypress tests
- Fixed modal cookie acceptance on cypress tests
- Lint retry popover system tests

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ Cypress.Commands.add('clearCookieConsentModal', () => {
   if (consentModal.length > 0) {
     cy.get('.truste_popframe').then((iframe) => {
       const body = iframe.contents().find('body');
-      cy.wrap(body).find('a.call').click()
+      cy.wrap(body).find('a.call').click().wait(2000)
```
Comment (de1987) [inline cypress/support/commands.js:41]: modal cookie agreement was being accepted, it was an intermittent issue when running cypress tests

File: src/Routes/Devices/RetryUpdatePopover.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ const popoverDescription = (reason, status, lastSeen) => (
     {reason === FAILURE
       ? 'The playbook failed to run. You can retry the update or build a new one.'
       : reason === TIMEOUT
-      ? 'The service timed out during the last update. You can retry the update'
+      ? 'The service timed out during the last update. You can retry the update or build a new one.'
```
Comment (de1987) [inline src/Routes/Devices/RetryUpdatePopover.js:27]: minor text fixes as described in the mockups.

**Review by tpapaioa — CHANGES_REQUESTED** [review]
Please put explanatory comments in the code instead of in PR comments.
Also, what is the reason for setting the lint config to remove semicolons?

**Comment by de1987** [comment]
> Please put explanatory comments in the code instead of in PR comments. Also, what is the reason for setting the lint config to remove semicolons?

It's done.

It's removing the semicolons is only for cypress tests code, and I added it because the old cypress tests are without semicolons.
@matoval @resoluteCoder explained we don't need to use semicolons for javascript, the idea here it's to keep the code uniform.

**Comment by de1987** [comment]
@tpapaioa done

**Review by tpapaioa — APPROVED** [review]

**Review by resoluteCoder — APPROVED** [review]

**Pull request merged**